lib/streamlit/__init__.py

```diff
@@ -343,18 +343,14 @@ def write(*args, **kwargs):
        height: 200px
 
     """
-    # Python2 doesn't support this syntax
```

Comment:
Note this comment tells us that we can now change the signature of this function to `def write(*args, unsafe_allow_html=False):`. That will break callers that were previously relying on the `**kwargs` argument to pass other arguments (e.g., a typo like `unsafe_alow_html=...`), but I think that's an okay break to make. I think we should take advantage of this (or, we could leave this comment here with "Python 2 didn't support..." and follow up later). Thoughts?

Reply:
Yeah that didn't seem to make sense based on the signature of st.markdown being invoked, which is the only item that was affected by this clause.

In terms of protecting the user from a misspelled keyword argument, yeah, maybe that's a thing we should do there.  I'll return to this section of the code today.

Reply:
Just wanted to check if you had made a determination here. To restate, I believe this comment was explaining why the signature for this function, `write()`, took a `kwargs` param when none seems to be needed (i.e., Python didn't allow explicit keyword args after a `*args` param).